proposals/0029-cooperative-vector.md

```diff
@@ -488,25 +449,25 @@ implementations can enable fast paths by allowing vectors to cooperate behind th
 fully occupied waves and uniform values for Matrix, Matrix Offset, Matrix Interpretation, Matrix Layout, Matrix Stride,
 Matrix Transpose and Bias, Bias Offset, Bias Interpretation, but this is not a requirement for functionality.
 
-### Shade Stages
+### Shader Stages
 
 The vector-matrix intrinsics are expected to be supported in all shader stages.
 
-// XXX TODO: Add query to determine which shader stages support these intrinsics.
+> TODO: Add query to determine which shader stages support these intrinsics.
```

Comment:
Given that we're saying they're supported on all shader stages, do we still need a query?

Reply:
I added that todo, based on an earlier review comment from you. We can remove it, if you are ok with specifying that it is suppoted in all stages.